etc/inc/allow-ssh.inc

```diff
@@ -2,6 +2,8 @@
 # Persistent customizations should go in a .local file.
 include allow-ssh.local
 
+# Ansible
+noblacklist ${HOME}/.ansible/cp
```

Comment:
Never blacklisted.

Reply:
Could you expand on that a little bit? For example, why is `.ssh` inside, the same logic should apply to `.ssh`. Please note that this file is included by another one, which means that not all blacklists are in this file.

Reply:
No file contains a blacklist for `.ansible/cp`, therefore it does not make sense to nobalcklist it. It is a no-op.

`.ssh` is blacklisted (in contrast to `.ansible/cp`), see
https://github.com/netblue30/firejail/blob/a4eb84d5e8165586a59d5e8e30c771fb7f133e2f/etc/inc/disable-common.inc#L444

Having the blacklist and noblacklist in the same file would be nonsense anyway and should never be the case.

Where is the blacklist

Reply:
Maybe check `grep ansible ~/.config/firejail/*`.

Reply:
Ahh, I see. But somehow accessing this file with firejail seems to be problematic. Ansible with SSH does not work at least.

Reply:
You can use `--debug` to geht more information. Maybe a symlink? And use firemon to see which sandboxes are started.